### PR TITLE
[MIG][12.0] website_sale_product_brand: Migration to v12.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT20150
Please @Tardo @chienandalu review.
One question @pedrobaeza, what's happend with the ACL in this module... ???

### CODE REVIEW

**Review by pedrobaeza — COMMENTED** [review]
You have made something weird squashing commits, so that the migration commit appears as done by the bot, but others commit that you can squash are not:

Selección_010

File: website_sale_product_brand/security/ir.model.access.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_product_brand_public,public.product.brand,product_brand.model_product_brand,base.group_public,1,0,0,0
```
Comment (pedrobaeza) [inline website_sale_product_brand/security/ir.model.access.csv:2]: You should remove this ACL as well as we have seen.

File: website_sale_product_brand/README.rst [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+To install this module, you need to install following module:
+->https://github.com/OCA/product-attribute/tree/11.0/product_brand
```
Comment (chienandalu) [inline website_sale_product_brand/README.rst]:
```suggestion
->https://github.com/OCA/product-attribute/tree/12.0/product_brand
```

**Review by chienandalu — APPROVED** [review]
:+1: Tested. Just a little issue and once @pedrobaeza comments are attended is ok for me

**Comment by sergio-teruel** [comment]
Changes done!!

**Comment by pedrobaeza** [comment]
The only weird thing right now is the squashing that has attribute incorrectly the migration to the OCA bot.

**Review by Tardo — APPROVED** [review]
:+1:

**Comment by pedrobaeza** [comment]
@sergio-teruel are you going to fix the authorship in the migration commit?

**Comment by sergio-teruel** [comment]
Ready!!

**Pull request merged**